### PR TITLE
[FEATURE] Automatic inclusion of PageTsConfig of extensions

### PR DESCRIPTION
**Pull request by linawolf**
https://docs.typo3.org/c/typo3/cms-core/main/en-us/Changelog/12.0/Feature-96614-AutomaticInclusionOfPageTsConfigOfExtensions.html

refs https://github.com/TYPO3-Documentation/TYPO3CMS-Reference-CoreApi/issues/1624

### CODE REVIEW

File: Documentation/ContentMapping/Index.rst [hunk 1]
```diff
@@ -40,19 +40,33 @@ Dynamic Content Rendering in TypoScript
 
 .. highlight:: typoscript
 
-Create a new directory :file:`Configuration/TsConfig/Page/` and inside this
-directory, a new file called :file:`Page.tsconfig` with the following
-content::
+Create a file at :file:`Configuration/page.tsconfig` the following
```
Comment (brotkrueml) [inline Documentation/ContentMapping/Index.rst:43]: **with** the following...

**Comment by linawolf** [comment]
@brotkrueml can we get it merged like this?

**Comment by brotkrueml** [comment]
> @brotkrueml can we get it merged like this?

Of course, didn't block the merge, just pointing to a mistake ;-)

File: Documentation/ContentMapping/Index.rst [hunk 2]
```diff
@@ -40,19 +40,33 @@ Dynamic Content Rendering in TypoScript
...
+
+.. note::
+   Starting with TYPO3 12 the file :file:`Configuration/page.tsconfig` is
+   automatically loaded during build time. See
+   :ref:`t3tsconfig:setting-page-tsconfig` on how to load the file in older
+   TYPO3 versions.
+   
+When there is only one site in your installation or all sites share the same 
+configurations we suggest to include this file globally as decribed here. 
+See :ref:`t3tsconfig:setting-page-tsconfig` on how to load the page TSconfig
```
Comment (susannemoog) [inline Documentation/ContentMapping/Index.rst:61]: missing "for a"

File: Documentation/CodeSnippets/ExtensionConfiguration/TcaOverridePages.rst.txt [hunk 3]
```diff
@@ -6,19 +6,6 @@
 
    <?php
    defined('TYPO3') || die();
-   
+
```
Comment (susannemoog) [inline Documentation/CodeSnippets/ExtensionConfiguration/TcaOverridePages.rst.txt:9]: the file is now functionally empty and can be removed.

**Review by susannemoog — CHANGES_REQUESTED** [review]

File: Documentation/ContentMapping/Index.rst (hunk 2 above)
Comment (linawolf) [reply Documentation/ContentMapping/Index.rst:61]: Wher is a "for a" missing?

File: Documentation/CodeSnippets/ExtensionConfiguration/TcaOverridePages.rst.txt (hunk 3 above)
Comment (linawolf) [reply Documentation/CodeSnippets/ExtensionConfiguration/TcaOverridePages.rst.txt]: done

**Review by mbrodala — APPROVED** [review]

**Pull request merged**